manage.py

```diff
@@ -1,9 +1,45 @@
-#!/usr/bin/env python
 """Django's command-line utility for administrative tasks."""
-import os
 import sys
+import os
+from contextlib import nullcontext, contextmanager
+
+
+@contextmanager
+def coverage_context():
+    """Context manager to collect code coverage data.
+
+    Uses coverage package to collect coverage, generate reports
+    and print coverage percentage.
+    """
+    # pylint: disable-next=import-outside-toplevel
+    from coverage import Coverage
+
+    cov = Coverage()
+    cov.erase()
+    with cov.collect():
+        yield
+    cov.save()
+    print(f'Coverage is {cov.report()}%')  # xml, html
```

Comment:
The comment is pointless here, because it doesn't mean anything, nor is related to printing out the text on this line.
```suggestion
    print(f'Coverage is {cov.report()}%')
```

Reply:
Agree